packages/cli/src/commands/index.ts

```diff
@@ -311,6 +311,7 @@ export function registerCommands(program: Command) {
       '--pattern <glob>',
       'Override glob for matching packages to upgrade',
     )
+    .option('--file <file>', 'Use local manifest file instead of fetching it')
```

Comment:
I'm thinking that rather than adding a separate option, we allow manifests files to be passed straight to `--release` instead, e.g. `--release ./release-manifest-1.7.json`. If the release ends with `.json` we check if the file exists, and if it does we `fs.readJson` and use it as the manifest.

It's a bit magic but I'm hoping it's still relatively intuitive without polluting the API. It also works pretty well if we for example want to add support for reading remote URLs in the future.